ui/components/app/fee-details-component/fee-details-component.js

```diff
@@ -29,9 +29,8 @@ import { useGasFeeContext } from '../../../contexts/gasFee';
 
```

Comment:
[nit][out of scope] 
we can remove the empty tag <></> around the template

Reply:
I would avoid changing existing code.

Reply:
ah right, this is a bug not a feature. would be nice to follow up on this